packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -127,21 +120,10 @@ export class OrgAuthInfo {
     });
   }
 
-  public static async getOrgApiVersion() {
-    const defaultUsernameOrAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
-      false
-    );
-    if (!defaultUsernameOrAlias) {
-      return undefined;
-    }
-    const username = defaultUsernameOrAlias
-      ? await AuthUtil.getInstance().getUsername(defaultUsernameOrAlias)
-      : undefined;
-    const connection = await Connection.create({
-      authInfo: await AuthInfo.create({ username })
-    });
+  public static async getOrgApiVersion(): Promise<string | undefined> {
+    const connection = await workspaceContext.getConnection();
     const apiVersion = connection.getApiVersion();
-    return apiVersion;
+    return (apiVersion as string) || undefined;
```

Comment:
nit: this line is kinda tricky.  Like I had to think kind of hard if we were gonna end up with 'undefined'. 
What do you think of 
```
return apiVersion ? String(apiVersion) : undefined;
```
In an ideal world I'd like throw a specific error if there is no apiVersion set that we could catch and handle from the caller but get the scope creep that'd introduce.